### DISCUSSION TITLE
npm install 报错，前几个月没问题，版本升级不兼容了？

### DISCUSSION THREAD

**Discussion by JorjanLLT** [ant-design/ant-design#31581] [Q&A]
![7290E4EB-C67A-4682-831A-95E06B64357C](https://user-images.githubusercontent.com/24427853/127453473-59e62f22-e7a7-4f93-8932-5a421e8f507c.png)
这些兼容弄得头疼

**Comment by songshuai10** [comment]
同样遇到这个问题了

**Comment by bfrontend** [comment]
遇到同样的问题， 怎么解决的啊

**Reply by JorjanLLT** [reply]
我把npm版本回退了好几个才勉强可以的。
使用的node版本为v14.16.0(npm:v6.14.11)
我当时用这个版本可以成功弄完
